docs/cse/introduction-to-cloud-siem.md

```diff
@@ -1,432 +0,0 @@
----
-id: introduction-to-cloud-siem
-title: Introduction to Cloud SIEM
-sidebar_label: Introduction to Cloud SIEM
-description: Learn basic concepts about Cloud SIEM.
----
-
-import useBaseUrl from '@docusaurus/useBaseUrl';
-
-Cloud SIEM is a cloud-based enterprise-grade security information and event management (SIEM) system. Cloud SIEM leverages Sumo Logic's core functionality, including data collection, ingestion, storage, and threat intelligence. Cloud SIEM is a purchased add-on with an ever-expanding library of content designed for security operations.
-
-Watch the following micro lesson to learn how to get started using Cloud SIEM for threat investigation.
-
-<Iframe url="https://www.youtube.com/embed/cDUOzQ63zmc?rel=0"
```

Comment:
@jpipkin1 I see this micro-lesson has not added to any of the new pages created, can we add this to the index page of Get Started with Cloud SIEM?

Reply:
@JV0812, I removed that micro lesson because the UI in it is very out-of-date. I'll add the micro lesson back after it is updated.